lib/fog/libvirt/compute.rb

```diff
@@ -57,12 +57,16 @@ def initialize(options={})
           require 'libvirt'
         end
 
-        private
-
         def client
-          return @client if defined?(@client)
+          @client ||= Class.new do
+            def libversion
+              1002009
```

Comment:
Would I be wrong in thinking in terms of Shindo tests this is the Mock class to be used by the tests?

I'll see if I can do anything else with it, maybe try setting it from within the test code itself.